src/proteomes/components/styles/busco.module.scss

```diff
@@ -0,0 +1,67 @@
+:root {
+  --s-color: #bce875; // completeSingle
+  --d-color: #0ba5a6; // completeDuplicated
+  --f-color: #eecba4; // fragmented
+  --m-color: #314bb4; // missing
```

Comment:
Ah, just a note, if this is set on `:root` maybe use a more specific name, containing `busco` for example, as this will be global for the whole document and it might be used elsewhere.
Otherwise if it's all in the same file you could do:
```css
.busco-view, .busco-legend-item {
  --s-color: #bce875; // completeSingle
  --d-color: #0ba5a6; // completeDuplicated
  --f-color: #eecba4; // fragmented
  --m-color: #314bb4; // missing
}
```
and this way they will be narrowly scoped